src/sanitizer.js

```diff
@@ -109,11 +107,29 @@ const WHITELISTED_ATTRS = [
   'on',
   'placeholder',
   'option',
-  /* Attributes added for amp-bind */
-  // TODO(kmh287): Add more whitelisted attributes for bind?
-  'text',
+  'submit-success',
+  'submit-error',
 ];
 
+/** @const {!Object<string, !Array<string>>} */
+const WHITELISTED_ATTRS_BY_TAGS = {
+  'amp-bind': [
```

Comment:
I'm not sure moving the `amp-bind` `[text]` attribute here makes sense, since there is never an `<amp-bind>` tag